src/super_gradients/training/transforms/keypoints/keypoints_pad_if_needed.py

```diff
@@ -0,0 +1,99 @@
+from typing import List, Iterable
+
+import cv2
+import numpy as np
+
+from super_gradients.common.object_names import Transforms, Processings
+from super_gradients.common.registry.registry import register_transform
+from super_gradients.training.samples import PoseEstimationSample
+from .abstract_keypoints_transform import AbstractKeypointTransform
+
+
+@register_transform(Transforms.KeypointsPadIfNeeded)
+class KeypointsPadIfNeeded(AbstractKeypointTransform):
+    """
+    Pad image and mask to ensure that resulting image size is not less than `output_size` (rows, cols).
+    Image and mask padded from right and bottom, thus joints remains unchanged.
+    """
+
+    def __init__(self, min_height: int, min_width: int, image_pad_value: int, mask_pad_value: float, padding_mode: str = "bottom_right"):
```

Comment:
I just learned recently that python includes `Literal` typing which may be appropriate here

```python
from typing import Literal


padding_mode: Literal["bottom_right", "center"] = "bottom_right"
```

I dont know if you ever used it or have an opinion on its usage, but to me it sounds like a good fit for this situation, it would add explicitness + type checking

Reply:
Actually, it only exists since `python==3.8`, and if I'm not wrong we support `python==3.7`, so that's a bad idea

Reply:
Yeah, unfortunately 3.7 is still there :/

Reply:
Well, string enums is still a thing